.github/workflows/docker-publish.yml

```
@@ -0,0 +1,64 @@
+
+      - name: Build and push Docker image
+        uses: docker/build-push-action@v5
+        with:
+          context: .
+          platforms: linux/amd64,linux/arm64
+          push: ${{ github.event_name != 'pull_request' }}
+          tags: ${{ steps.meta.outputs.tags }}
+          labels: ${{ steps.meta.outputs.labels }}
+          cache-from: type=gha
+          cache-to: type=gha,mode=max
+
```

YAMLlint (1.38.0)

[error] too many blank lines (1 > 0)

(empty-lines)